components/slider/slider.css

```diff
@@ -0,0 +1,8 @@
+.slider-container {
+	text-align: center;
+}
+
+.slider {
+	width: 50%;
+	cursor: pointer;
```

Comment:
suggestion: keep slider width 100% so that while importing it into ecommerce it would be easier to adjust it's width

Reply:
Thank you for suggestion